pkg/lang/csharp/queries/structural/property_declarations.scm

```diff
@@ -0,0 +1,10 @@
+(property_declaration
+  type: (_) @type
+  [accessors:
+    (accessor_list
+      (accessor_declaration) ? @get (#match? @get "^get")
```

Comment:
nit: does this need to match `^get$` (with the `$` at the end) or is that not necessary

Reply:
Not crucial, but definitely more technically correct. This and a few other structural queries aren't actually used yet, so they're subject to revision (or removal if we feel strongly about it).